.circleci/config.yml

```diff
@@ -524,6 +524,33 @@ jobs:
           command: |
             make schemas_check
 
+  schemas_deploy:
+    docker:
+      - image: cimg/python:3.10.8 # poetry 1.2.2
+    steps:
+      - checkout
+      - run:
+          name: Check for package version change in last commit before proceeding.
+          command: |
+            if git diff main HEAD~1 schemas/pyproject.toml | grep 'version'
```

Comment:
Just curious why `HEAD~1` and not `HEAD`?

Reply:
`HEAD` is the current commit so `HEAD~1` compares current to previous.